codalab/apps/web/bundles.py

```diff
@@ -212,7 +212,7 @@ def upload_bundle(self, source_file, bundle_type, worksheet_uuid):
             '''
             # Construct info for creating the bundle.
             bundle_subclass = get_bundle_subclass(bundle_type) # program or data
-            metadata = metadata_util.request_missing_metadata(bundle_subclass, {}, initial_metadata={'name': source_file.name, 'description': 'Upload ' + source_file.name})
+            metadata = metadata_util.request_missing_metadata(bundle_subclass, {}, initial_metadata={'name': source_file.name, 'description': 'Upload ' + source_file.name}, bypass_editor=True)
```

Comment:
Let's split request_missing_metadata into two functions, one for filling in default values, and one for actually requesting missing values.  Then, here we should just call the former.  In bundle_cli.py, we should call possibly both functions.